tests/runtimes/simple-keyvalue/api/src/api.rs

```diff
@@ -9,6 +9,9 @@ pub struct KeyValue {
 }
 
 runtime_api! {
+    //  Gets runtime ID of the runtime.
+    pub fn get_remote_runtime_id(()) -> Option<String>;
```

Comment:
Why "remote"? Everything is remote :-)

Reply:
I was mixing the `get_runtime_id()` of the protocol and this one, so I deliberately name it _remote. But it's better to stay consistent, yeah.